tests/createOption.test.ts

```diff
@@ -70,32 +69,44 @@ describe("Option Creation", () => {
         .signers([authorityKeypair])
         .rpc();
 
-      expect(optionCreatedEvent.poolAccount).to.eql(poolAccountKey);
-      expect(optionCreatedEvent.option).to.eql(optionTwoAccountKey);
-      expect(optionCreatedEvent.title).to.eql(optionTitle);
+      expect(optionCreatedEvent.poolAccount).toEqual(poolAccountKey);
+      expect(optionCreatedEvent.option).toEqual(optionTwoAccountKey);
+      expect(optionCreatedEvent.title).toEqual(optionTitle);
     } catch (e) {
-      expect(e.message).to.include("PoolOptionDoesNotMatchHash");
+      expect(e.message).toContain("PoolOptionDoesNotMatchHash");
```

Comment:
Should we now move to `.toThrow`?

Reply:
Yess, we would be moving away from the try-catch itself. I left out those changes on purpose, to keep the PR lean. There's some other changes with less verbose event listeners as well, that I had in mind. Should I include in the same PR?